test/e2e/specs/all-features-confirmation/recordset/presentation.spec.js

```diff
@@ -416,6 +502,100 @@ describe('View recordset,', function() {
                 });
             });
 
+            it("apply sorting on different columns.", function () {
+                browser.get(browser.params.url + "/recordset/#" + browser.params.catalogId + "/product-recordset:" + accommodationParams.table_name + "?limit=3");
+                var EC = protractor.ExpectedConditions;
+                chaisePage.waitForElementInverse(element.all(by.id("spinner")).get(0));
+                browser.wait(EC.presenceOf(chaisePage.recordsetPage.getRows().get(2)), browser.params.defaultTimeout);
+                for (var j = 0; j < accommodationParams.sortedData.length; j++) {
+                    (function (k) {
+                        //Click on sort button
+                        chaisePage.recordsetPage.getColumnSortButton(accommodationParams.sortedData[k].columnPosition).click().then(function () {
+                            browser.wait(EC.textToBePresentInElement(element(by.id('rs-total-count')),"Displaying 3 of 5 Records"), browser.params.defaultTimeout);
+                            return chaisePage.waitForElementInverse(element.all(by.id("spinner")).get(0));
+                        }).then(function () {
+                            // Check if the url has @sort by column name
```

Comment:
After checking if the url has the right `@sort identifier`, add an expectation to verify the proper glyphicon is shown in the column header. This will be relevant below as well for the descending case.